release/Magarena/scripts/The_Locust_God.groovy

```diff
@@ -0,0 +1,41 @@
+def DelayedTrigger = {
+    final MagicSource staleSource, final MagicPlayer stalePlayer, final MagicCard refCard ->
```

Comment:
same problem, ref card should not be used directly. It should be renamed staleCard.